src/app/view/components/dot-relationship-tree/dot-relationship-tree.component.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { DebugElement } from '@angular/core';
```

Comment:
should we create a host component to make the tests, instead of assign the props directly like this `component.velocityVar` ?

Reply:
I was advised to assign props directly.